### DISCUSSION TITLE
'copilot is encountering temporary issues' happens again

### DISCUSSION THREAD

**Discussion by mo-bai** [community/community#27983] [Copilot Conversations]
The error occurred two days ago and has not been resolved yet ! I want to know the reason and when it will be solved .

Copilot version: v1.36.6340
vs-code version: 1.69.2

**Comment by MrSarno** [comment]
There definitely hasn’t been a global outage for two full consecutive days.

Have you tried removing the extension, restarting your computer, and performing a fresh install of the extension? Which OS do you use?

**Reply by mo-bai** [reply]
Update plugins, uninstall plugins and reinstall, restart vs-code, restart mac, change network. All of these have been tried, but none of them work. Waiting for loading for a long time will prompt 'copilot is encountering temporary issues'

**Comment by davecheney** [comment]
@mo-bai I don't see any requests from your userid. Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Reply by mo-bai** [reply]
I find . My request for 'https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex' failed, connect ETIMEDOUT 20.84.233.211:443 . maybe I should change my proxy ! Thanks for your reply ！

**Comment by lusains** [comment]
Copilot version: v1.1.31.1908
idea version: 2021.3.3 
The error occurred every time , I ried removing the extension, restarting ,but did not work at all~  and i am a paying user

**Comment by rahijoshi** [comment]
This is happening to me too. I installed copilot yesterday and it's been giving that error ever since.

**Reply by davecheney** [reply]
i'm sorry you are having difficulties. Can you please tell me a bit more about what you tried, and what happened. Thank you.

**Reply by rahijoshi** [reply]
Sure, thanks for the fast response! I tried uninstalling and reinstalling the extension. Tried restarting my laptop and installing the extension. It's giving the same error.
In the output section of vscode, I'm seeing 
`[ERROR] [default] [2022-09-01T21:00:05.787Z] Request Error: self signed certificate in certificate chain
[ERROR] [ghostText] [2022-09-01T21:00:05.788Z] Error on ghost text request FetchError: self signed certificate in certificate chain`

**Reply by davecheney** [reply]
@rahijoshi please see https://github.com/orgs/community/discussions/29127

**Reply by rahijoshi** [reply]
Thank you! It worked :D The error went away
